backend/appwrite-functions/set-wallet/main.py

```diff
@@ -8,38 +10,47 @@
 PAYLOAD = os.environ.get("APPWRITE_FUNCTION_DATA")
 APPWRITE_PROJECT = os.environ.get("APPWRITE_FUNCTION_PROJECT_ID")
 
-# provided by cloud function env vars
+# provided by appwrite function env vars
 APPWRITE_ENDPOINT = os.environ.get("APPWRITE_ENDPOINT")
 APPWRITE_API_KEY = os.environ.get("APPWRITE_API_KEY")
 WALLETS_COLLECTION_ID = os.environ.get("WALLETS_COLLECTION_ID")
 
+# instantiate database connection
```

Comment:
I see the good intent but personally, I find comments like these counter-productive. Same for the added comments below. Of course, depends on your decision, but I wouldn't do that.

The professional software-engineering way of explaining code is by outlining code with good function names. My personal experience is that additional comments disturb the readability, particularly if the code is clear, but outlining them into functions is better (more levels of detail).

The function would be called  `instantiateDatabaseConnection()`.